service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/AbstractEmailContentProvider.java

```diff
@@ -14,8 +14,8 @@
 @SuppressWarnings("VariableDeclarationUsageDistance")
 public abstract class AbstractEmailContentProvider {
 
-    private static final String ORDER_KEY = "orders";
-    private static final String DIRECTIONS_KEY = "directionsAndInterim";
+    private static final String ORDER_OPTION_KEY = "orders_option";
```

Comment:
I would just use the string values directly. I had to extract them to a common variable before because sonar scan was complaining. Dont think you will have the same issue.